js/constants/constants.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
     ERROR_SCREEN: "errorScreen"
   },
 
+  CUSTOM_EVENTS: {
```

Comment:
I think INITIAL_PLAY_REQUESTED is fine. This gets triggered when the user touches/clicks the screen, which is a user request for initial playback.